core/orchestrator.go

```diff
@@ -447,7 +447,7 @@ func (n *LivepeerNode) getSegmentChan(md *SegTranscodingMetadata) (SegmentChan,
 	if len(n.SegmentChans) >= MaxSessions {
 		return nil, ErrOrchCap
 	}
-	sc := make(SegmentChan, 1)
+	sc := make(SegmentChan, 4)
```

Comment:
Can we define a constant for this so its clear that its an important parameter that controls the maximum # of segments that can be queued on an orchestrators?